### PR TITLE
ui: Statements tracing - Diagnostics tab with empty state

### LINKED ISSUE #45245: ui: add tracing tab to statement details page
Design link: https://www.figma.com/file/C85d0mrz8dkNs7Nw5mcTmf/obsrv_admin-ui-statements?node-id=1235%3A82

To support trace and EXPLAIN collection in the UI, we will need to add a new tab to the statement details page labeled "Diagnostics". This issue tracks:
- creating the tab
- creating the empty state for the page

### PR DESCRIPTION
**Pull request by koorosh**
Resolves: #45245 

Note: Current changes is not connected to Statement Details page and not visible on UI yet.
It is safe to merge without exposing partially implemented functionality.

New tab is added with a state when no diagnostics is available.

Screenshot 2020-02-21 at 14 49 26

### CODE REVIEW

File: pkg/ui/src/components/link/link.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+// licenses/APL.txt.
+
+import React from "react";
+import "./link.styl";
+
+interface LinkProps {
+  href: string;
+  target?: "_blank" | "_parent" | "_self";
+}
+export function Link(props: React.PropsWithChildren<LinkProps>) {
```
Comment (dhartunian) [inline pkg/ui/src/components/link/link.tsx]: Can we call this something other than Link to disambiguate from Router Link?

**Review by dhartunian — APPROVED** [review]
LGTM, just one naming thing I'd like to change. Thanks!

File: pkg/ui/src/components/link/link.tsx (hunk 1 above)
Comment (koorosh) [reply pkg/ui/src/components/link/link.tsx]: Done, renamed to `Anchor`.

**Comment by koorosh** [comment]
Outdated

**Pull request closed without merging**